snapcraft/cli/version.py

```diff
@@ -0,0 +1,36 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import click
+
+import snapcraft
+
+
+@click.group()
+def versioncli():
+    """Version commands"""
+    pass
+
+
+@versioncli.command('version')
+def version():
+    """Obtain snapcraft's version number.
+
+    Examples:
+        snapcraft version
+        snapcraft --version
+    """
+    click.echo('snapcraft, version {}'.format(snapcraft.__version__))
```

Comment:
add a constant here with the version message, make it look like the current on in click, from the source:
```
'%(prog)s, version %(version)s'
```

Reply:
I had to hardcode "snapcraft, version %(version)" otherwise it would produce issues.

Thank you for your review. Hopefully I did not mess up commit after a long time (had a couple of busy weeks). If you prefer, I can start over. to avoid multiple commits and such.